src/main/java/http/controller/AbstractController.java

```diff
@@ -0,0 +1,27 @@
+package http.controller;
+
+import http.HttpMethod;
+import http.HttpRequest;
+import http.Response.HttpResponse;
+
+/**
+ * Created by iltaek on 2020/06/09 Blog : http://blog.iltaek.me Github : http://github.com/iltaek
+ */
+public abstract class AbstractController implements Controller{
```

Comment:
공백, 공백라인 컨벤션 등의 부분들은 습관적으로 반영하시는 것이 좋다고 생각해요 😊
```suggestion
public abstract class AbstractController implements Controller {
```